cni/deployments/kubernetes/Dockerfile.install-cni

```diff
@@ -9,7 +9,11 @@ ARG ISTIO_BASE_REGISTRY=gcr.io/istio-release
 FROM ${ISTIO_BASE_REGISTRY}/base:${BASE_VERSION} as debug
 
 # The following section is used as base image if BASE_DISTRIBUTION=distroless
-FROM ${ISTIO_BASE_REGISTRY}/distroless:${BASE_VERSION} as distroless
+# This image is a custom built debian11 distroless image with multiarchitecture support.
+# It is built on the base distroless image, with iptables binary and libraries added
+# The source can be found at https://github.com/istio/distroless/tree/iptables
+# This version is from commit 86c4972a9f5f245cfb382c8e1e95f176d968c882.
+FROM ${ISTIO_BASE_REGISTRY}/iptables@sha256:87e7a5b30a2844e16363cdd3b6ff06565079627a592c95f5c0fc9aaa56366452 as distroless
```

Comment:
Is there a plan to use debian12? Not sure if it's using nftables and you'd have to install iptables-legacy